seatunnel-engine/seatunnel-engine-server/src/main/java/org/apache/seatunnel/engine/server/master/JobMaster.java

```diff
@@ -333,6 +361,30 @@ public void run() {
         }
     }
 
+    public static void handleSaveMode(SeaTunnelSink sink) {
+        if (sink instanceof SupportSaveMode) {
+            Optional<SaveModeHandler> saveModeHandler =
+                    ((SupportSaveMode) sink).getSaveModeHandler();
+            if (saveModeHandler.isPresent()) {
+                try (SaveModeHandler handler = saveModeHandler.get()) {
+                    new SaveModeExecuteWrapper(handler).execute();
+                } catch (Exception e) {
+                    throw new SeaTunnelRuntimeException(HANDLE_SAVE_MODE_FAILED, e);
+                }
+            }
+        } else if (sink.getClass()
+                .getName()
+                .equals(
+                        "org.apache.seatunnel.connectors.seatunnel.common.multitablesink.MultiTableSink")) {
+            // TODO we should not use class name to judge the sink type
```

Comment:
Please explain the detailed reason. I see that it is possible to implement `SaveModeHandler`

Reply:
It is difficult to perform some system-level operations at the connector level, such as create savemode-related events. I will solve this problem in the next PR.